src/main/drivers/timer_hal.c

```diff
@@ -339,7 +343,11 @@ void configTimeBase(TIM_TypeDef *tim, uint16_t period, uint32_t hz)
     timerHandle[timerIndex].Handle.Init.RepetitionCounter = 0x0000;
 
     HAL_TIM_Base_Init(&timerHandle[timerIndex].Handle);
-    if (tim == TIM1 || tim == TIM2 || tim == TIM3 || tim == TIM4 || tim == TIM5 || tim == TIM8 || tim == TIM9) {
```

Comment:
(it may be possible to use timerIndex and mask ... but considerable macro magic will be necessary)